src/cli-storage.ts

```diff
@@ -0,0 +1,19 @@
+import fs from "node:fs";
+import os from "node:os";
+
+const __confDir = os.homedir() + "/.baseplate";
+if (!fs.existsSync(__confDir)) {
+  fs.mkdirSync(__confDir);
+}
+
+export async function storageProvider() {
+  // @ts-ignore
+  const conf = await import("conf");
+  return new conf.default({
+    projectName: "baseplate.cloud",
+    configName: ".bpc",
+    fileExtension: "",
+    cwd: __confDir,
+    encryptionKey: "910b01ee3a166b1132a01735ef4e695",
```

Comment:
Following up on the conversation thread at https://github.com/ConvexCooperative/baseplate-cli/pull/30#discussion_r1565136445 - is hardcoding this a temporary thing? When will it be removed/changed/fixed?

Reply:
this could implemented as part of #29